### PR TITLE
Facemesh update to SDK

### PR DESCRIPTION
**Pull request by Erol444**
Update after releasing SDK `1.10.2` (or `1.11`, not sure how the SDK number will increase after current `1.10.1`)

### CODE REVIEW

**Comment by Erol444** [comment]
https://github.com/luxonis/depthai-experiments/assets/18037362/04d594e2-2238-49d6-a818-a9079270beff

File: gen2-facemesh/requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 opencv-python
 numpy
-depthai==2.16.0.0
\ No newline at end of file
+depthai-sdk==1.9.6
```
Comment (mhilden) [inline gen2-facemesh/requirements.txt:3]: @Erol444 Hi - I'd like to test this PR, but I can't find the tag for `depthai-sdk==1.9.6` (I can see `1.9.5` in the `depthai` repo). Can you please help me locate the `v1.9.6` version of the `depthai-sdk`?

Comment (Erol444) [reply gen2-facemesh/requirements.txt:3]: Hi @mhilden , should be actually 1.10.1, let me try it out and merge it if it works

Comment (Erol444) [reply gen2-facemesh/requirements.txt:3]: Nope, we'll need to wait for the new release of depthai-sdk.. facemesh NN support is currently in `develop`

Comment (mhilden) [reply gen2-facemesh/requirements.txt:3]: Okay, @Erol444; thanks for the help. I'll wait to test this until the lastest `depthai-sdk` is out and hopefully by then I can test it on our new OAK-D SR PoE.

Comment (Erol444) [reply gen2-facemesh/requirements.txt:3]: doesn't work atm, I'm just fixing it up..

Comment (Erol444) [reply gen2-facemesh/requirements.txt:3]: @mhilden  this should now fix it: https://github.com/luxonis/depthai/commit/5863140708b25ceac03fac1662f685bb59fef479

**Comment by mhilden** [comment]
@Erol444 Thanks for the updates. I was able to get this to work by:
 - Creating a new Python 3.9 `venv` environment
 - Switching my local copy of `depthai` to the `develop` branch
 - Changing the version number of the local `develop` branch to `1.10.1b0` and installing locally from source (`pip install -e .`)
 - Installing all dependencies for `depthai` via `pip install -r requirements.txt`
 - Changing the installation requirement for the `gen2-facemesh/requirements.txt` to `1.10.1b0`
 - Running the `depthai-experiments/gen2-facemesh/main.py` file from my local `venv`

This does seem to be an improvement over the gen1 iteration of facemesh; thanks for doing the work to update it and for the fresh example.

**Comment by Erol444** [comment]
Hi @mhilden , thanks for testing and confirming it works as expected!

**Pull request merged**